src/app/chat/view.nim

```diff
@@ -19,6 +20,10 @@ QtObject:
       callResult: string
       messageList: Table[string, ChatMessageList]
       activeChannel*: ChatItemView
+      activeStickerPackId*: int
+      mStickerPacks*: StickerPackList
+      mStickers*: Table[int, StickerList]
```

Comment:
Had that at first, but changed them as I as pulling my hair out debugging the issue with `rowCount`. I thought that maybe there was a naming collision (has happened before with Nim) with the `ChatsView.stickers` property and `ChatsView.stickers` `QtProperty`), so prefixed the property names with `m` to avoid collision.